common/src/project.rs

```diff
@@ -124,6 +144,8 @@ pub mod tests {
             ".invalid",
             "invalid.name",
             "invalid.name.",
+            "test-fuck-fuck",
```

Comment:
consider changing to `crap` as it is still profanity and little less abrasive :smiley: